packages/expo-cli/src/exp.ts

```diff
@@ -364,9 +365,15 @@ function runAsync(programName: string) {
     });
 
     program.on('command:*', subCommand => {
-      log.warn(
-        `"${subCommand}" is not an ${programName} command. See "${programName} --help" for the full list of commands.`
+      let msg = `"${subCommand}" is not an expo command. See "expo --help" for the full list of commands.`;
+      const availableCommands = program.commands.map(cmd => cmd._name);
+      const suggestion = availableCommands.find(
+        cmd => leven(cmd, subCommand[0]) < cmd.length * 0.4
```

Comment:
what's the logic behind `cmd.length * 0.4`? might be useful to add in a comment

Reply:
`leven` returns the distance between 2 given strings based on the `Levenshtein distance algorithm`. Here we're setting the threshold value to `40%` of the command length.

Reply:
right, any reason why you chose 40% as the threshold or just a guess at what might work well?

Reply:
https://github.com/dcporter/didyoumean.js/#threshold

Reply:
CI is red for an unknown reason 🤔